Firmware/temperature.cpp

```diff
@@ -1484,32 +1504,24 @@ void min_temp_error(uint8_t e) {
 }
 
 void bed_max_temp_error(void) {
-#if HEATER_BED_PIN > -1
-  //WRITE(HEATER_BED_PIN, 0);
-#endif
   if(IsStopped() == false) {
     SERIAL_ERROR_START;
-    SERIAL_ERRORLNPGM("Temperature heated bed switched off. MAXTEMP triggered !");
+    SERIAL_ERRORLNPGM("Heaters switched off. MAXTEMP BED triggered !");
```

Comment:
Can we build these strings from parameters? There are multiple instances of them and only MAXTEMP/ MINTEMP BED/AMBIENT actually change.